packages/cdk-environment-resources/lib/index.ts

```diff
@@ -60,16 +61,38 @@ export class CdkEnvironmentResources extends cdk.Construct {
             groupMetrics: [autoscaling.GroupMetrics.all()]
         });
 
-        if(props.userData){
+        if (props.userData) {
             autoScalingGroup.addUserData(props.userData);
         }
 
         cdk.Tags.of(autoScalingGroup).add("EC2Group", "ecs-container-instance", {
             applyToLaunchedInstances: true
         });
 
-        const targetTrackingScalingPolicy = autoScalingGroup.scaleOnCpuUtilization("ScalingPolicy", {
-            targetUtilizationPercent: 50,
+        const memoryReservationMetric = new cloudwatch.Metric({
```

Comment:
In case you hadn't seen this, there is a method that returns this construct automatically for you: https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ecs.Cluster.html#metricwbrmemorywbrreservationprops